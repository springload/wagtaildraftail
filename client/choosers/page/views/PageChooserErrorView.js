import React from 'react';
import PropTypes from 'prop-types';

const propTypes = {
  errorMessage: PropTypes.node,
};

const defaultProps = {
  errorMessage: null,
};

const PageChooserErrorView = ({ errorMessage }) =>
  <div className="nice-padding">
    <div className="help-block help-critical">
      {errorMessage}
    </div>
  </div>;

PageChooserErrorView.propTypes = propTypes;
PageChooserErrorView.defaultProps = defaultProps;

export default PageChooserErrorView;
