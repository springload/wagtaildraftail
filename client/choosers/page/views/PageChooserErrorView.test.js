import React from 'react';
import { shallow } from 'enzyme';
import PageChooserErrorView from './PageChooserErrorView';

describe('PageChooserErrorView', () => {
  it('renders', () => {
    expect(shallow((
      <PageChooserErrorView errorMessage="test" />
    ))).toMatchSnapshot();
  });
});
