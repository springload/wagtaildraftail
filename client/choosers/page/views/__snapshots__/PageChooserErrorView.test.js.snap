// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageChooserErrorView renders 1`] = `
<div
  className="nice-padding"
>
  <div
    className="help-block help-critical"
  >
    test
  </div>
</div>
`;
